### PR TITLE
Use 'kubectl' instead of 'tkn' for tree, some minor refactoring

### PR DESCRIPTION
**Pull request by evidolob**
This PR increase Tekton tree loading speed. Also now Tekton tree use only `kubectl` to get data for nodes.
Also it add better update tree when pipelinerun is running.

Demo:
ezgif com-video-to-gif (3)

Fix: #240
Fix: #239

### CODE REVIEW

File: src/tkn.ts [hunk 1]
```diff
@@ -692,29 +682,21 @@ export class TknImpl implements Tkn {
...
     return pipelineTree;
-
   }
 
-  async refreshPipelineRun(pipelineName: string): Promise<void> {
-    await kubectl.watchPipelineRun(pipelineName);
-    pipelineExplorer.refresh();
+  async refreshPipelineRun(pipeline: TektonNode): Promise<void> {
+    await kubectl.watchPipelineRun(pipeline.getName(), () => {
+      pipelineExplorer.refresh(pipeline);
```
Comment (sudhirverma) [inline src/tkn.ts:690]: Screenshot 2020-04-17 at 1 53 42 PM

When pipelineRun finish its execution it doesn't refresh the separate tree of pipelineRun and TaskRun

**Comment by sudhirverma** [comment]
And also I get the error while automatic refreshing the tree 

Screenshot 2020-04-17 at 2 17 03 PM

File: src/tkn.ts [hunk 2]
```diff
@@ -195,7 +195,7 @@ export class Command {
   }
   @verbose
   static listPipelineRuns(name: string): CliCommand {
-    return newTknCommand('pipelinerun', 'list', name, '-o', 'json');
+    return newK8sCommand('get', 'pipelinerun', '-l', `tekton.dev/pipeline=${name}`, '-o', 'json');
```
Comment (sudhirverma) [inline src/tkn.ts:198]: can we show the running pipelineRun and taskRun at the top? I think when we were using `tkn` command that time we were not facing that problem.

Screenshot 2020-04-17 at 2 31 28 PM

File: src/tkn.ts [hunk 3]
```diff
@@ -269,20 +274,20 @@ export class Command {
...
   static listTaskRun(): CliCommand {
-    return newTknCommand('taskrun', 'list', '-o', 'json');
+    return newK8sCommand('get', 'taskrun', '-o', 'json');
   }
   static listConditions(): CliCommand {
     return newK8sCommand('get', 'conditions', '-o', 'json');
   }
   static listPipelineRun(): CliCommand {
-    return newTknCommand('pipelinerun', 'list', '-o', 'json');
+    return newK8sCommand('get', 'pipelinerun', '-o', 'json');
```
Comment (sudhirverma) [inline src/tkn.ts:290]: can we show the running pipelineRun and taskRun at the top? I think when we were using tkn command that time we were not facing this problem.

Screenshot 2020-04-17 at 2 31 28 PM

**Comment by evidolob** [comment]
> And also I get the error while automatic refreshing the tree
> 
> Screenshot 2020-04-17 at 2 17 03 PM

@sudhirverma  I don't see that error, can you share the pipeline?

**Comment by sudhirverma** [comment]
I think we should refresh the separate node of `taskRun` as we are refreshing the `taskRun` under `pipelineRun`.

Screenshot 2020-04-21 at 5 28 39 PM

**Pull request merged**